finetrainers/args.py

```diff
@@ -337,6 +337,7 @@ class Args:
     validation_every_n_steps: Optional[int] = None
     enable_model_cpu_offload: bool = False
     validation_frame_rate: int = 25
+    do_not_run_validation: bool = False
```

Comment:
Otherwise, 

https://github.com/a-r-r-o-w/finetrainers/blob/da9d7d92a5e71aec82b7a1a25d97631458212bd5/finetrainers/args.py#L486

will lead to an error.

Reply:
I am an idiot. 

8841dbd